defaults/main.yml

```diff
@@ -46,6 +46,10 @@ bbb_freeswitch_ioschedule_realtime: true
 bbb_freeswitch_ipv6: true
 bbb_freeswitch_external_ip: "stun:{{ (bbb_stun_servers | first).server }}"
 
+bbb_kurento_interfaces:
```

Comment:
Hmm, i would like to have a default, that does not change things. Could you maybe change that?

Reply:
Sure I could. I was having the problem with the default configuration that kurento was advertising the docker ip addresses on webrtc. This is a problem because the the client tries to connect to those ip addresses, fails, it takes time etc. I thought that defaulting to the default interface and lo would be a reasonable compromise, but I do see the point that it could break things. Maybe alternatively iterate over all interfaces and just filter out the docker interfaces?

Reply:
I would rather prefer the manual way (e.g. an empty default) to avoid accidentally breaking thing and rather having a good explanation (with an example) in the readme.

Reply:
Ok, I understand the rationale behind that. I will do that